readme.md

```diff
@@ -35,6 +35,7 @@ General purpose icons used everywhere.
 - [Octicons](https://github.com/primer/octicons#readme) - A scalable set of icons handcrafted with <3 by GitHub. ([Website](https://octicons.github.com))
 - [Open Iconic](https://github.com/iconic/open-iconic#readme) - Open Iconic is the open source sibling of Iconic. ([Website](https://useiconic.com/open))
 - [Picon](https://github.com/yne/picon#readme) - Small ligature-based icon font and SVG. ([Website](https://yne.fr/picon))
+- [Radix Icons](https://icons.modulz.app/) - A crisp set of 15×15 icons designed by the Modulz team.
```

Comment:
Please GitHub link first, followed by the website at the end, like the rest of the entries.